_posts/2019-06-26-xbps-news.md

```diff
@@ -0,0 +1,58 @@
+---
+title: News about XBPS
+layout: post
+---
+
+In the last week we pushed a lot of XBPS updates, **a lot**.
+
+The previous post about XBPS 0.54 was rather short and I would like to show a few more new things.
+If you are interested in all changes, read the [NEWS](https://github.com/void-linux/xbps/blob/master/NEWS) file in the void-linux/xbps repository.
+
+## Performance improvements
+
+Some users might have noticed that removing the `plasma-desktop` package took ages, because of some dependency cycles.
+The dependency resolution algorithm has been completely rewritten and is now able to resolve the dependencies of `kde-plasma` with
```

Comment:
hmm looks incomplete?

Reply:
oh yes I didn't know what to write and skipped it and then forgot about it.
"within seconds", "in no time".

Reply:
Looks ok to me, albeit some grammar issues.